target/linux/ramips/image/mt7621.mk

```diff
@@ -335,6 +335,17 @@ define Device/iodata_wnpr2600g
 endef
 TARGET_DEVICES += iodata_wnpr2600g
 
+define Device/iptime_a8004t
+  MTK_SOC := mt7621
+  IMAGE_SIZE := 16128k
+  UIMAGE_NAME := a8004t
+  DEVICE_VENDOR := ipTIME
+  DEVICE_MODEL := A8004T
+  DEVICE_PACKAGES := \
+	kmod-mt7615e kmod-usb3 kmod-usb-ledtrig-usbport wpad-basic
```

Comment:
This device does not have dedicated USB LED, so kmod-usb-ledtrig-usbport does not need to be included by default.

Reply:
Aha I forgot

Reply:
Fixed.